request_public_data.py

```diff
@@ -45,9 +47,9 @@
 
 for index, entry_list in enumerate(phone_list):
 	if index < 4:
-		logging.debug("iphone" + str(index%4+1) + " first entry:")
+		logging.debug("iphone" + str(index+1) + " first entry:")
 	else:
-		logging.debug("android" + str(index%4+1) + " first entry:")
+		logging.debug("android" + str(index-3) + " first entry:")
```

Comment:
it's `index-3` because this for loop is iterating through the concatenated list of iphone and android, so the index for android starts at 4